src/crate/theme/rtd/crate/partials/_head_css_variables.html

```diff
@@ -13,16 +13,21 @@
 
 <style>
   body {
-    {{ declare_css_variables(theme_light_css_variables, furo_pygments.light) }}
+    --color-code-background: #f8f8f8;
+    --color-code-foreground: black;
   }
+
   @media not print {
     body[data-theme="dark"] {
-      {{ declare_css_variables(theme_dark_css_variables, furo_pygments.dark) }}
+      --color-code-background: #202020;
+      --color-code-foreground: #d0d0d0;
     }
     @media (prefers-color-scheme: dark) {
       body:not([data-theme="light"]) {
-        {{ declare_css_variables(theme_dark_css_variables, furo_pygments.dark) }}
+        --color-code-background: #202020;
+        --color-code-foreground: #d0d0d0;
       }
     }
   }
+
```

Comment:
It looks like variables have been used before here, but now you are using hard-coded values. Is it no longer possible to use variables going forward?

Reply:
If it is no longer possible, can we still define the colors in a different location, for example in a `custom.scss`, if we have it already, instead of within an `.html` file?

Reply:
@amotl Well the thing is, those variables only work if we import a whole lot more config stuff. This file originates from https://github.com/pradyunsg/furo/blob/main/src/furo/theme/furo/partials/_head_css_variables.html and only work if we include the bulk of https://github.com/pradyunsg/furo/blob/main/src/furo/__init__.py - which I don't have the resources to look at these days. So adding thise two variables manually was the next best thing until one of those cold winter nights arrive to do it ;)

Reply:
Thank you for clarifying!